src/mobile/src/ui/components/SeedVaultExportComponent.js

```diff
@@ -20,6 +20,7 @@ import InfoBox from './InfoBox';
 import Button from './Button';
 import CustomTextInput from './CustomTextInput';
 import PasswordFields from './PasswordFields';
+import { removeNonAlphaNumeric } from 'shared-modules/libs/utils';
```

Comment:
The CI is failing with:
> $ ./node_modules/.bin/eslint src/mobile/
/Users/vagrant/git/src/mobile/src/ui/components/SeedVaultExportComponent.js
  23:39  error  Absolute imports should come before relative imports  import/first
✖ 1 problem (1 error, 0 warnings)

My bad, I should have told you beforehand. To make ESLint happy, you'll have to move this above the relative imports:

```js
import { isAndroid, getAndroidFileSystemPermissions } from 'libs/device';
import { removeNonAlphaNumeric } from 'shared-modules/libs/utils'; <--
import InfoBox from './InfoBox';
import Button from './Button';
import CustomTextInput from './CustomTextInput';
import PasswordFields from './PasswordFields';
```

Once that's done, we should be good to go